docs/en/Contributing-Getting-Started.md

```diff
@@ -267,6 +271,10 @@ into the Alluxio code base.
 
 Congratulations! You have successfully contributed to Alluxio! Thank you for joining the community!
 
+## <a name="video"></a>Video
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/QsbM804rc6Y" frameborder="0" allowfullscreen></iframe>
```

Comment:
Not yet, although I could probably add a jekyll plugin that would give us that. Is that preferable?